tests/Unit/InboundEmailTest.php

```
@@ -0,0 +1,158 @@
+	// Ticket-ID parser  /\[TKT-(\d+)\]/i
+	// -------------------------------------------------------------------------
+
+	/**
+	 * [TKT-XXXX] at the end of a subject line is extracted correctly.
+	 */
+	public function test_parse_ticket_id_from_subject(): void {
+		$subject = 'Re: Your support request [TKT-1234]';
+		$matched = preg_match( '/\[TKT-(\d+)\]/i', $subject, $matches );
+		$this->assertSame( 1, $matched );
+		$this->assertSame( '1234', $matches[1] );
+	}
```

PHPMD (2.15.0)

[error] The method test_parse_ticket_id_from_subject is not named in camelCase. (undefined)

(CamelCaseMethodName)